http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorSupervisorSpec.scala

```diff
@@ -3,7 +3,7 @@ package org.broadinstitute.dsde.workbench.leonardo.monitor
 import java.time.Instant
```

Comment:
I didn't actually change anything functionally in this file, but I happened to open it by accident before ClusterMonitorSpec, and had to refactor because I had no idea what it was trying to test with all the duplication